docs/command-center.md

```diff
@@ -0,0 +1,33 @@
+# Command Center
+
+Opsdroid Command Center is the dashboard that allows you to interact with Opsdroid without the need to have any connector installed (other than the default websocket one). The dashboard allows you to enable/disable modules as well see your active modules and installed configuration from within your browser.
+
+Opsdroid Command Center is powered by the [opsdroid web](https://github.com/opsdroid/opsdroid-web) package, you need to first install the package using `npm`.
+
+## Configuration
+
+Opsdroid Command Center configuration lives under the `web` section of Opsdroid configuration.
+
+```yaml
+web:
+  command-center:
+    enabled: true
+    enable-dashboard: true
+    dashboard-port: 8081
+    token: <Your chosen secret token>
+    excluded-keys:
+      - module
+      - path
+```
+
+Let's have a look at each of these configuration arguments.
+
+- `enabled` - Toggles Command Centter on/off, if enabled new routes will be added to Opsdroid's [RESTful api](rest-api.md)
```

Comment:
I think it would be fine to enable everything by default and maybe not even have a disable.

Reply:
I think we should keep it disabled and only enable it if users want to. the reason for that is  because anyone with the URL would be able to deactivate/activate modules if they know opsdroid's url 🤔